suite-native/atoms/src/TransactionItem/TransactionItem.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Icon } from '@trezor/icons';
+import { capitalizeFirstLetter } from '@trezor/utils';
+
+import { Box } from '../Box';
+import { Text } from '../Text';
+
+type TransactionStatus = 'received' | 'sent';
+
+type TransactionItemProps = {
+    cryptoCurrency: string;
+    amountInCrypto: number;
+    amountInFiat: number;
+    transactionStatus: TransactionStatus;
+};
+
+const transactionItemWrapperStyle = prepareNativeStyle(utils => ({
+    paddingVertical: utils.spacings.medium,
+    height: 66,
+    flexDirection: 'row',
+    alignItems: 'center',
+}));
+
+const iconWrapperStyle = prepareNativeStyle(utils => ({
+    width: 32,
+    height: 32,
+    justifyContent: 'center',
+    alignItems: 'center',
+    marginRight: utils.spacings.small,
+    backgroundColor: utils.colors.gray300,
+    borderRadius: utils.borders.radii.round,
+}));
+
+const transactionSignStyle = prepareNativeStyle<{ isReceivedTransaction: boolean }>(
+    (utils, { isReceivedTransaction }) => ({
+        ...utils.typography.hint,
+        marginRight: utils.spacings.small,
+        color: isReceivedTransaction ? utils.colors.forest : utils.colors.red,
+    }),
+);
+
+const FIAT_CURRENCY = 'Kc'; // Note: This will probably change and will be passed from selector
+
+export const TransactionItem = ({
+    cryptoCurrency,
+    amountInFiat,
+    amountInCrypto,
+    transactionStatus,
+}: TransactionItemProps) => {
+    const { applyStyle } = useNativeStyles();
+    const isReceivedTransaction = transactionStatus === 'received';
```

Comment:
I would prefer creating some object with possible statuses and their possible sign:
```tsx
const transactionStatusSigns: Record<TransactionsStatus, string> = {
	received: "+",
	send: "-"
}
```

it will be also more flexible if we add for example pending state in future